uk.co.compucorp.civicrm.hrcore/backstop_data/casper_scripts/ssp-leave-absences/staff/my-leave-calendar_tooltip.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var page = require('../../../page-objects/ssp-leave-absences-my-leave-calendar');
+
+module.exports = function (casper) {
+  page.init(casper)
+    .clearAllSelectedMonths()
```

Comment:
Why do we need to clear all months, then show February in Particular for the year 2016.

Reply:
> Why do we need to clear all months

Because the current month is selected by default.

> then show February in Particular for the year 2016.

Because the sample data creates a leave request for the staff on February 2016. By checking this month and year we avoid adding new requirements for BackstopJS